Configuration/PyReleaseValidation/python/relval_2017.py

```diff
@@ -30,7 +30,7 @@
 #        (Patatrack HCAL-only: TTbar - on CPU)
 #   2021 (DD4HEP: TTbar, ZMM)
 #        (ele guns 10, 35, 1000; pho guns 10, 35; mu guns 1, 10, 100, 1000, QCD 3TeV, QCD Flat)
-#        (ZMM, TTbar, ZEE, MinBias, TTbar PU, TTbar PU premix, ZEE PU, TTbar design)
+#        (ZMM, TTbar, ZEE, MinBias, TTbar PU, TTbar PU premix, ZEE PU,GluGluTo2Jets, TTbar design,GluGluTo2Jets)
```

Comment:
sorry for being a bit picky but could you update to 
(ZMM, TTbar, ZEE, MinBias, TTbar PU, TTbar PU premix, ZEE PU, TTbar design, GluGluTo2Jets, GluGluTo2Jets PU) note for the space as well

Reply:
Don't be sorry, that's ok, thank you... I put that way because I imagined it should have a correspondence with the wf id, as in line 60, where I put them ordered by their number. Tell me if that is not the case. Thank you.